dockerfiles/stages/4-production

```diff
@@ -18,20 +18,30 @@ ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update -y \
   && apt-get install -y \
     apt-utils \
+    apt-transport-https \
     curl \
+    wget \
     ca-certificates \
     dnsutils \
     dumb-init \
     gettext \
+    gnupg2 \
+    lsb-release \
     libgmp10 \
     libgomp1 \
     libssl1.1 \
-    postgresql \
-    postgresql-contrib \
     tzdata \
     jq \
     sudo
 
+# --- Install Postgresql 13
+RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add - \
```

Comment:
not a blocker but to not introduce extra dependencies I'd use `curl -s https://www.postgresql.org/media/keys/ACCC4CF8.asc` instead of wget

Reply:
Good catch @simisimis .
I'll update the PR and resubmit it for review

Reply:
@simisimis 
changed from `wget` to `curl` and used `$deb_codename` variable to define the postgresql repo list